src/app/shared/forms/planet-tag-input.component.ts

```diff
@@ -211,15 +213,19 @@ export class PlanetTagInputComponent implements ControlValueAccessor, OnInit, On
     });
   }
 
-  dialogTagUpdate(tag, isSelected, tagOne = false) {
+  dialogTagUpdate(tag, isSelected, tagOne = false, selectedTags) {
```

Comment:
`selectedTags` is not used as a function parameter, so you should declare it as a variable with `let` instead of as a function parameter.

Reply:
Reading this a little more I see `selectedTags` is only modified, it doesn't actually do anything.  Did you mean to declare `selectedTags` as a property of this component instead of **planetTagInputDialogComponent**?

Reply:
This is also causing the funky behavior.  It is never defined, so `selectedTags.push` throws an error

Reply:
@paulbert Looks like I accidentally deleted some of my code while merging. I'll keep you posted. The funky behavior is with the blue buttons, not the issue at hand.